debezium-connector-oracle/pom.xml

```diff
@@ -26,7 +26,7 @@
             <scope>provided</scope>
         </dependency>
         <dependency>
-            <groupId>com.oracle.instantclient</groupId>
+            <groupId>com.oracle.database.jdbc</groupId>
```

Comment:
To be sure: it won't end up in the dist TAR.GZ, right?

Reply:
Tested, it is not there.